.ci/azure-pipelines.yml

```diff
@@ -1,43 +1,36 @@
+# Universal Windows Platform
+# Build a Universal Windows Platform project using Visual Studio.
+# Add steps that test and distribute an app, save build artifacts, and more:
+# https://aka.ms/yaml
+
 trigger:
-  batch: true
-  branches:
-    include:
-    - '*'
-  tags:
-    include:
-    - '*'
+- master
 
-pr:
-  branches:
-    include:
-    - '*'
```

Comment:
We use these triggers for all projects so PRs and any branches will also get checked, can we revert this change?

Reply:
Reverted